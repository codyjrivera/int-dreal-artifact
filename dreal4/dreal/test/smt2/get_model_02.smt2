(declare-const x Int)
(declare-const y Real)
(assert (= x 10))
(assert (= y (+ x 3)))
(assert (> x y))
(check-sat)
(get-model)
(exit)
