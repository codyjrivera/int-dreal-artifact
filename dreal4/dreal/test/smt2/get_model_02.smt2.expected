unsat
(error "model is not available")
